server/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -1041,4 +1042,63 @@ public void test_update_sys_tables_returning_values_with_expressions_and_outputn
             assertThat(response.rows()[i][2], is(true));
         }
     }
+
+    @Test
```

Comment:
Isn't the code logic/path the same for partitions? If so I'd suggest to remove one test.

Reply:
Yes thank you, will remove one. I blindly followed the same pattern as the column position fix.